src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -81,6 +82,34 @@ class Task extends React.Component<Props, State> {
         modalError: undefined,
         isEditing: false,
         isLoading: false,
+        isAssigneeListOpen: false,
+    };
+
+    componentDidUpdate(prevProps: Props) {
+        // reset state to props assigned_to
+        const { assigned_to } = this.props;
+        const { assigned_to: prevAssignedTo } = prevProps;
+        if (prevAssignedTo !== assigned_to) {
+            this.setState({
+                assigned_to,
+            });
+        }
+    }
+
+    handleShowMoreAssignees = () => {
+        const { assigned_to } = this.state;
+
+        if (assigned_to.next_marker) {
+            this.fetchTaskCollaborators().then(() => {
```

Comment:
Is a `.catch` handler needed?

Reply:
added a noop for the rejected callback - the error state is handled in the feed api class